apps/chef_test/src/chef_test_suite_helper.erl

```diff
@@ -32,10 +34,29 @@
          make_id/1,
          make_az_id/1,
          actor_id/0,
+
+         make_orgs/0,
+         org_name/0,
+         other_org_name/0,
+
          the_org_id/0,
-         other_org_id/0
+         other_org_id/0,
+
+         create_record/1,
+         list_records/1,
+         update_record/1,
+         fetch_record/1,
+         delete_record/1
         ]).
 
+-define(ORG_AUTHZ_ID, <<"10000000000000000000000000000002">>).
+-define(ORG_NAME, <<"testorg">>).
+
+-define(OTHER_ORG_AUTHZ_ID, <<"99999999999999999999999999999999">>).
```

Comment:
I've been looking at the authz id stuff in a few places - because it's mocked out, we can pretty much just use the same authz id everywhere and stop worrying about making them unique  Not sure that this is something to worry about now, though, so much as a general cleanup we can apply.